### PR TITLE
fix: maven compiler in pom

### PR DESCRIPTION
**Pull request by Ankit098**

### CODE REVIEW

**Review by kamal-kaur04 — APPROVED** [review]

**Review by sreenivas-k — APPROVED** [review]
Thank you

File: pom.xml [hunk 1]
```diff
@@ -21,6 +21,8 @@
 		<cucumber.version>1.1.2</cucumber.version>
 		<selenium.version>2.31.0</selenium.version>
 		<junit.version>5.0</junit.version>
+		<maven.compiler.source>8</maven.compiler.source>
```
Comment (karanshah-browserstack) [inline pom.xml]: This should be 1.8 right?

Comment (Ankit098) [reply pom.xml]: Either one should work.. will update.

**Review by sreenivas-k — APPROVED** [review]
Thanks Ankit!

**Pull request merged**